### DISCUSSION TITLE
Unable to start chia

### DISCUSSION THREAD

**Discussion by jatdaluram** [Chia-Network/chia-blockchain#6915] [Support]
Chia is not starting even after deleting the wallet db files. 
Have latest version of chia windows client.
also deleted wallet data base file

**Comment by loppefaaret** [comment]
under the folder `C:\Users\<USERNAME>\.chia\mainnet\log\` there is a file called `debug.log` which might shed some light as to what errors are happening when you try to start Chia.
if you keep having issues, perhaps try and swing by our Keybase chat, and either send a message in out #support chat, or ding @pyl directly
https://keybase.io/team/chia_network.public

**Comment by BrandtH22** [comment]
@jatdaluram , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
